docs/backport-workflow.md

```diff
@@ -0,0 +1,526 @@
+# Automated Backport Workflow
+
+This document describes the automated backport workflow for the LND project.
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [How to Use](#how-to-use)
+3. [Workflow Triggers](#workflow-triggers)
+4. [Label Format](#label-format)
+5. [Workflow Steps](#workflow-steps)
+6. [Handling Conflicts](#handling-conflicts)
+7. [Multiple Backports](#multiple-backports)
+8. [Technical Details](#technical-details)
+9. [Troubleshooting](#troubleshooting)
+
+## Overview
+
+The automated backport workflow simplifies the process of backporting merged PRs from the `master` branch to release branches (e.g., `v0.20.x-branch`, `v0.19.x-branch`).
+
+Instead of manually creating branches, cherry-picking commits, and creating PRs, maintainers can simply add a label to the master PR, and the workflow handles the rest.
+
+## How to Use
+
+### Basic Usage
+
+1. **Merge a PR to master** (or have it already merged)
+2. **Add a backport label** in the format: `backport-v<version>-branch`
+   - Example: `backport-v0.20.x-branch`
+3. **The workflow automatically**:
+   - Validates the target branch exists
+   - Cherry-picks the commits
```

Comment:
Why do we need to cherry pick the commits now? Are we taking all the commits except the changes to the release notes?